diff --git a/node_modules/cpy/index.js b/node_modules/cpy/index.js
index 5e3ac83..3b39bb2 100644
--- a/node_modules/cpy/index.js
+++ b/node_modules/cpy/index.js
@@ -60,7 +60,7 @@ const preprocessDestinationPath = (source, destination, options) => {
 };
 
 module.exports = (source, destination, {
-	concurrency = (os.cpus().length || 1) * 2,
+	concurrency = (os.cpus().length || 1) * 8,
 	...options
 } = {}) => {
 	const progressEmitter = new EventEmitter();
