diff --git a/node_modules/electron-prompt/lib/index.js b/node_modules/electron-prompt/lib/index.js
index 871b75f..c5caf23 100644
--- a/node_modules/electron-prompt/lib/index.js
+++ b/node_modules/electron-prompt/lib/index.js
@@ -1,7 +1,7 @@
 const electron = require('electron');
-
-const BrowserWindow = electron.BrowserWindow || electron.remote.BrowserWindow;
-const ipcMain = electron.ipcMain || electron.remote.ipcMain;
+const remote = require("@electron/remote");
+const BrowserWindow = electron.BrowserWindow || remote.BrowserWindow;
+const ipcMain = electron.ipcMain || remote.ipcMain;
 const url = require('url');
 const path = require('path');
 
